Sprint-3/implement/is-proper-fraction.js

```diff
@@ -32,3 +34,28 @@
 // target output: false
 // Explanation: The fraction 3/3 is not a proper fraction because the numerator is equal to the denominator. The function should return false.
 // These acceptance criteria cover a range of scenarios to ensure that the isProperFraction function handles both proper and improper fractions correctly and handles potential errors such as a zero denominator.
+//CHecked
+
+function isProperFraction(numerator, denominator){
+    if(denominator === 0){
+        return false;
+    }
+    else if(numerator === denominator){
```

Comment:
Should it only return 'false' if the numerator is _equal to_ the denominator?

Reply:
No, it should return false in the case of zero denominator as well as in cases where the numerator is greater than or equal to the denominator because both represent invalid or improper fractions in this context.